src/main/resources/com/microsoft/jenkins/containeragents/aci/AciCloud/help-logAnalyticsCredentialsId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Choose a <b>Username with password</b>  credential. The username is the workspace ID and the password is the workspace key.
+</div>
```

Comment:
Finding the workspace key is not easy, let's make it easier for the user, how does this look to you?

```suggestion
<p>
    Choose a <strong>Username with password</strong> credential. The username is the workspace ID and the password is the workspace key.
</p>
<p>
    You can use the <a href="https://docs.microsoft.com/en-us/cli/azure/what-is-azure-cli">Azure CLI</a> to retrieve the ID and key.
    Replace the <code>$resourceGroup</code> and <code>$workspaceName</code> placeholders with the required values.
</p>
<p>
    ID: <code>az monitor log-analytics workspace show -g $resourceGroup --workspace-name $workspaceName --query customerId -o tsv</code>
</p>
<p>
    Key: <code>az monitor log-analytics workspace get-shared-keys -g $resourceGroup --workspace-name $workspaceName --query primarySharedKey -o tsv</code>
</p>

```